examples/custom-session-az-swa/frontend/src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
Do we need the `testing-library` parts?

Reply:
You’re right, it was just part of the create-react-app template. I've removed it.